### PR TITLE
update docs needed per open-source guidelines

### PR DESCRIPTION
**Pull request by anniehu4**
### Summary:

Since this project is already public, we should bring it up to the [CZI open-source guidelines](https://czi.atlassian.net/wiki/spaces/TECH/pages/1900972990/Creating+an+Open+Source+Project+at+CZI#CreatinganOpenSourceProjectatCZI-Licensing). Mainly
- adding a SECURITY.md file
- adding a project status

### Test Plan:
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ The [Education Initiative](https://chanzuckerberg.com/education/)'s design syste
...
-Run linter and fix all fixable issues | `npm run lint:fix`
-Build packages | `npm run build`
-Run the component generator | `npx plop`
+| Description                           | Command                              |
+| ------------------------------------- | ------------------------------------ |
+| Install dependencies                  | `npm install && npx lerna bootstrap` |
+| Run linter                            | `npm run lint`                       |
+| Run linter and fix all fixable issues | `npm run lint:fix`                   |
+| Build packages                        | `npm run build`                      |
+| Run the component generator           | `npx plop`                           |
```
Comment (anniehu4) [inline README.md:22]: hmm my prettier did this, but doesn't look like it changed anything in the rich diff

Comment (dierat) [reply README.md:22]: Just easier to read the raw file I guess.

**Review by dierat — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Reporting Security Vulnerabilities
+
+The Chan Zuckerberg Initiative takes security issues seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
```
Comment (dierat) [inline SECURITY.md:3]: Nit: in order to use a contraction to connect two clauses, both clauses need to have a subject. `, and will make` -> `, and we will make`.

Comment (anniehu4) [reply SECURITY.md:3]: TIL, thanks 🤓

File: README.md (hunk 1 above)
Comment (ahuth) [reply README.md:22]: WAT I've never seen prettier change markdown tables before 🤔 

Kind of cool, though

**Pull request merged**